### ISSUE TITLE
Export to Excel changed after upgrade

### ISSUE THREAD

**Issue by servicedeskrumos** [kimai/kimai#521]
Hi guys,

I have upgraded Kimai to both latest stable (0.9.3.1384) and latest development (RC) versions and lost the ability to see the Comment column data in the exported file. Older version where this was working was 0.9.2.1306.

Basically the steps to reproduce the issue are:
Open Kimai and go to Export tab, then select User and its associated Project. Although I do not see data present in Comment column, the exported Excel file has its data.

Basically, I am using the Comment field to reference an internal Project ID and it is configured when editing a project and you go to Comment tab.

Can you please help?

Thank you!

Cheers,

Bruno Martins

**Comment by kevinpapst** [comment]
Hey Bruno / @servicedeskrumos .
I tried to reproduce the problem in the current master, but wasn't able to reproduce. 
I see the comments from each timesheet entry both, in the export view and the generated Excel file.

Could you please retest (we changed quite a lot since you opened the issue).
If you still experience the problem, let me know how to reproduce it. I do not use the comment column normally, so I just added some comments to timesheet entries to see what will happen.

**Comment by servicedeskrumos** [comment]
Hi @kevinpapst,
I've found that this was an additional feature added by an old programmer of my customer's company. Please forget this topic.

Thanks for your cooperation.
